src/lib/ui/noise-sensor/NoiseThresholdsList.tsx

```diff
@@ -0,0 +1,140 @@
+            tooltipContent={
+              device.device_type === 'minut_sensor' ? (
+                <div className='seam-detail-section-tooltip-inner-content'>
+                  <span className='seam-tooltip-content'>
+                    {t.minutTooltipFirst}
+                  </span>
+                  <span className='seam-tooltip-content'>
+                    {t.minutTooltipSecond}
+                  </span>
+                </div>
+              ) : (
+                t.tooltip
+              )
+            }
+          >
+            <Content
+              isInitialLoading={isInitialLoading}
+              noiseThresholds={noiseThresholds}
+            />
+          </DetailSection>
+
+          <div className='seam-detail-section-footer'>
+            <div className='seam-empty-div' />
+            <div className='seam-detail-section-footer-content'>
+              <div className='seam-detail-section-footer-content-text'>
+                <p>{getTimeZoneCaption(device, noiseThresholds)}</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </DetailSectionGroup>
+    </>
+  )
+}
+
+function Content({
+  isInitialLoading,
+  noiseThresholds,
+}: {
+  isInitialLoading: boolean
+  noiseThresholds: NoiseThresholds[] | undefined
+}): JSX.Element | JSX.Element[] {
+  if (isInitialLoading) {
+    return (
+      <DetailRow
+        label={<span className='seam-detail-row-empty-label'>{t.loading}</span>}
+      />
+    )
+  }
+
+  if (noiseThresholds == null || noiseThresholds.length === 0) {
+    return (
+      <DetailRow
+        label={<span className='seam-detail-row-empty-label'>{t.none}</span>}
+      />
+    )
+  }
+
+  return noiseThresholds?.map((noiseThreshold) => (
+    <DetailRow
+      key={noiseThreshold.noise_threshold_id}
+      label={
+        <div className='seam-detail-row-label-column'>
+          {noiseThreshold.name !== '' && (
+            <span className='seam-detail-row-label'>{noiseThreshold.name}</span>
+          )}
+          <div className='seam-detail-row-label-block'>
+            <span className='seam-row-sublabel seam-row-sublabel-text-default'>
+              {formatTime(noiseThreshold.starts_daily_at)}
+            </span>
+            <ArrowRightIcon />
+            <span className='seam-row-sublabel seam-row-sublabel-text-default'>
+              {formatTime(noiseThreshold.ends_daily_at)}
+            </span>
+          </div>
+        </div>
+      }
+    >
+      <p>
+        {noiseThreshold.noise_threshold_decibels} {t.decibel}
+      </p>
+    </DetailRow>
+  ))
+}
+
+const getTimeZoneCaption = (
+  device: NoiseSensorDevice,
+  thresholds: NoiseThresholds[] | undefined
+): string | null => {
+  if (device.location?.timezone != null) {
+    return `All times in ${formatTimeZone(device.location.timezone)}`
+  }
+
+  if (thresholds == null) {
+    return null
+  }
+
+  const zonedTime = ZonedTime.from(thresholds[0]?.starts_daily_at ?? '')
```

Comment:
I think ZonedTime will throw an error if you pass null or an empty string.
https://github.com/seamapi/zoned-time/blob/main/src/lib/parse-zoned-time-string.ts#L24-L32

This should be a type error, but if not it means there is an upstream type issue in zoned time. Feel free to PR upstream if you find it.

Reply:
Nope, all good—added an additional check on the first threshold to resolve this.